rfid_tracker/app.py

```diff
@@ -7,10 +7,13 @@
 import typer
 from mfrc522 import SimpleMFRC522
 from models import Action, Base, Member
+from slack_webhook import Slack
 
 MAX_SPOTS = 19
+WEBHOOK_URL = 'https://hooks.slack.com/services/xxx/yyy'
 
 app = typer.Typer()
+slack = Slack(url=WEBHOOK_URL)
```

Comment:
... use environment variables for it:

`os.environ["WEBHOOK_URL"]`

Reply:
Done